bitnami/apache/ci/static-analysis/.trivyignore

```diff
@@ -0,0 +1,18 @@
+# Default capabilities not dropped: avd.aquasec.com/appshield/ksv003
```

Comment:
I've created a new directory under `apache/ci/static-analysis` to hold not just the `.trivyignore` but future custom policy files.